INSTALL.rst

```diff
@@ -155,3 +175,29 @@ tests::
 
    export SKIMAGE_TEST_STRICT_WARNINGS=False
    pytest --pyargs skimage
+
+3. Linux operating system installation
+--------------------------------------
+We do not recommend using the opearting system's version of python for a few
+reasons:
+
+  1. It is easy to type ``sudo pip`` and cause major problems in the operating
+     system.
+  2. It is hard to control exactly what version of packages you may have
+     installed.
+  3. Not all packages are available from the system's package manager. 
+
+That said, if you must use the system environment, use the **system's** package
+manger (e.g. ``apt``, ``dnf``, ``yum``), and **not** ``pip`` to perform the
+installation.
+
+For example, on Ubuntu, typing::
+
+  sudo apt install python3-skimage
+
+Will perform a valid install of scikit-image on Ubuntu 19.10, but will install
+version ``0.14.2`` (even for Python 3!!) while the latest version of
+scikit-image at the time of writing is ``0.16.2``. Again, **never** type::
```

Comment:
I suggest removing specific version mentions, instead something like: "will perform a valid install of scikit-image on the system Python, but it will usually be a few major versions behind the latest release on PyPI."

Also, it's best not to sound very bossy, and instead make it strong recommendations:

"Again, we strongly recommend avoiding "sudo pip" or "sudo python -m pip""